C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -346,7 +362,7 @@ public async void ProcessActivityAsync(HttpRequestMessage request, HttpResponseM
                 return;
             }
 
-            if (!VerifySignatureAsync(request, response))
+            if (!VerifySignature(request, response))
```

Comment:
Is something missing here???

Reply:
@GasparAcevedoZainSouthworks  you mean, in the _'true'_ part of the **if**? There's nothing. I assume it is because the js code is still under construction. If you want, maybe I can reverse the condition and simply eliminate the empty half of the **if**